client/my-sites/exporter/post-type-options.jsx

```diff
@@ -12,18 +12,22 @@ import FormRadio from 'components/forms/form-radio';
 import Select from './select';
 import Label from 'components/forms/form-label';
 
-import { setPostType } from 'state/site-settings/exporter/actions';
+import { setPostType, setPostTypeFilters } from 'state/site-settings/exporter/actions';
 import {
 	getPostTypeOptions,
+	getPostTypeValues,
 	getSelectedPostType,
 } from 'state/site-settings/exporter/selectors';
 
 const mapStateToProps = ( state, ownProps ) => {
 	const siteId = state.ui.selectedSiteId;
 	const options = getPostTypeOptions( state, siteId, ownProps.postType );
+	const values = getPostTypeValues( state, siteId, ownProps.postType );
```

Comment:
Yeah, I've struggled with naming all these parts. Really it should be 'postTypeOptionsValues' which is a bit of a mouthful. It's the list of values selected for each option under the post type. So for example, say you select 'Uncategorized' in the 'Category' dropdown under 'Posts', that's stored as something like:

```js
post: { category: 1 }
```
So `getPostTypeValues( state, siteId, 'post' )` would return `{ category: 1 }`

Would `postTypeOptions` and `postTypeValues` for the props make sense?